internals/feature_helpers.py

```diff
@@ -476,6 +476,7 @@ def get_features_by_impl_status(limit: int | None=None, update_cache: bool=False
     feature_list = []
     for section in query_results:
       if len(section) > 0:
+        section = [f for f in section if not f.deleted]
         section = [f for f in section if f.feature_type != FEATURE_TYPE_ENTERPRISE_ID]
```

Comment:
Nit: these list comprehensions can be combined
```suggestion
        section = [f for f in section if not f.deleted]
        section = [f for f in section if not f.deleted
                   and f.feature_type != FEATURE_TYPE_ENTERPRISE_ID]
```

Reply:
True, but I kind of like having them separate.  Skipping deleted features seems like a separate idea than getting in to the details of the feature types.

Reply:
That's fine with me - no worries about making this change then. 🙂